proj/cmake/platform_linux.cmake

```diff
@@ -191,17 +191,9 @@ if( GSTREAMER_GL_INCLUDE_DIRS AND GSTREAMER_GL_LIBRARIES )
 endif()
 
 # Boost
-if( CINDER_BOOST_USE_SYSTEM )
-	find_package( Boost 1.54 REQUIRED COMPONENTS system filesystem )
-	list( APPEND CINDER_LIBS_DEPENDS ${Boost_LIBRARIES} )
-	list( APPEND CINDER_INCLUDE_SYSTEM_PRIVATE ${Boost_INCLUDE_DIRS} )
-else()
-	set( LINUX_LIB_DIRECTORY "${CINDER_PATH}/lib/linux/${CINDER_ARCH}/" )
-	list( APPEND CINDER_LIBS_DEPENDS
-		${LINUX_LIB_DIRECTORY}/libboost_system.a
-		${LINUX_LIB_DIRECTORY}/libboost_filesystem.a
-	)
-endif()
+find_package( Boost 1.54 REQUIRED COMPONENTS system filesystem )
+list( APPEND CINDER_LIBS_DEPENDS ${Boost_LIBRARIES} )
+list( APPEND CINDER_INCLUDE_SYSTEM_PRIVATE ${Boost_INCLUDE_DIRS} )
```

Comment:
Does this mean that we no longer be shipping any boost libraries on linux, in other words the `CINDER_BOOST_USE_SYSTEM` option is the only option? If so I'm not against it, just thought we had discussed switching the default behavior but still shipping prebuilt binaries. Maybe that has shown itself to be more trouble than its worth, though.

Reply:
My opinion would be to make the default and only behavior systemwide installed Boost. It has proven indeed problematic in the past even by just checking out the Boost submodule and then having mixed includes ( e.g. coming from Cinder's version and from a system wide installed version of Boost ) Not linking with the prebuilt binaries is not really enough in this case.